File: app/src/main/java/com/nanicky/devteam/main/playback/mediasource/MediaUpdateNotifier.java
package com.nanicky.devteam.main.playback.mediasource;

import com.nanicky.devteam.main.common.view.BaseMediaStoreViewModel;

public class MediaUpdateNotifier {
    private BaseMediaStoreViewModel baseMediaStoreViewModel;

    public BaseMediaStoreViewModel getBaseMediaStoreViewModel() {
        return baseMediaStoreViewModel;
    }

    public void setBaseMediaStoreViewModel(BaseMediaStoreViewModel baseMediaStoreViewModel) {
        this.baseMediaStoreViewModel = baseMediaStoreViewModel;
    }

    public void update() {
        if (baseMediaStoreViewModel!= null) baseMediaStoreViewModel.update();
    }
}

File: app/src/main/java/com/nanicky/devteam/main/playback/mediasource/MediaUpdateNotifier.kt
package com.nanicky.devteam.main.playback.mediasource

import com.nanicky.devteam.main.common.view.BaseMediaStoreViewModel

class MediaUpdateNotifier {
    var baseMediaStoreViewModel: BaseMediaStoreViewModel<*>? = null

    fun update() {
        if (baseMediaStoreViewModel != null) baseMediaStoreViewModel!!.update()
    }
}